Update grants.md (#67)
cpp-lln-lab/CPP_LabGuide: doc/grants.md

# Grants

[More ideas here](https://intranet.uclouvain.be/en/myucl/administrations/adre/research-funding-opportunities.html)

> Contact person at UCL can help a lot: A.D.

Reviewer: I think it is fine to put names, at least here these are public info and for us we will forget in a couple of days who A.D. is. It is probably better to not share emails (even though very likely public as well)

```suggestion
Contact person at UCL can help a lot: Anouk Distelman

```

Credits / money for PhD students is rolled over from one year to the next

## Travel

### FNRS

> 
> 
> 2 months in advance for within EU, 3 months for outside EU
> 
> 
> **don’t need to be FNRS to apply**
> 
> 
> Provide max. 2 conference grants per institute
> 
> [FNRS Mobilité](https://www.frs-fnrs.be/fr/financements/mobilite-monde)
> 
> **PLEASE NOTE they deadline is very strict! No possibility to apply if you miss the deadline**

Reviewer: ```suggestion

 - 2 months in advance for within EU, 3 months for outside EU
 - **don’t need to be FNRS to apply**
 - Provide max. 2 conference grants per institute

[FNRS Mobilité](https://www.frs-fnrs.be/fr/financements/mobilite-monde)

**PLEASE NOTE the deadline is very strict! No possibility to apply if you miss the deadline**

```

> ### FNRS
> 
> 
> 2 months in advance for within EU, 3 months for outside EU
> 
> 
> **don’t need to be FNRS to apply**
> 
> 
> Provide max. 2 conference grants per institute
> 
> [FNRS Mobilité](https://www.frs-fnrs.be/fr/financements/mobilite-monde)
> 
> **PLEASE NOTE they deadline is very strict! No possibility to apply if you miss the deadline**

Reviewer: ```suggestion
### FNRS

- 2 months in advance for within EU, 3 months for outside EU
- **don’t need to be FNRS to apply**
- Provide max. 2 conference grants per institute

[FNRS Mobilité](https://www.frs-fnrs.be/fr/financements/mobilite-monde)

**PLEASE NOTE they deadline is very strict! No possibility to apply if you miss the deadline**

```
